internal/guest/runtime/runc/container.go

```diff
@@ -74,7 +74,8 @@ func (c *container) ExecProcess(process *oci.Process, stdioSet *stdio.Connection
 	return p, nil
 }
 
-// Kill sends the specified signal to the container's init process.
+// Kill sends the specified signal to the container's init process. When syscall.SIGTERM or syscall.SIGKILL is sent,
+// send the specified signal to all processes inside the container
```

Comment:
https://github.com/opencontainers/runc/blob/main/kill.go#L28

runc kill --all is not supported. Why dont we just go back to signing just the init pid?